pages/add/form/detail-info-fields.js

```diff
@@ -80,15 +51,17 @@ let Tags = React.createClass({
     }).filter(suggestion => !!suggestion);
   },
   render: function() {
+    let delimiters = [9,13,32,188]; // keycodes for tab,enter,space,comma
```

Comment:
hey @Pomax , I found out the value returned from `charCodeAt` isn’t necessarily the same as the `keycode` returned from keyboard event. Comma is one example - it is `44` from `charCodeAt` but its keycode is `188`.  I searched it a bit but couldn't find a solution... do you know a way to convert character to keycode? If not I guess what I can do is to move that line from `render()` to the top of the file and make it a `const`.